### PR TITLE
COMP: Update GitHub Actions for v5.1.2, macOS 10.15 image updates

### PR DESCRIPTION
**Pull request by thewtex**

### CODE REVIEW

File: .github/workflows/build-test-package.yml [hunk 1]
```diff
@@ -39,6 +39,9 @@ jobs:
         python -m pip install --upgrade pip
         python -m pip install ninja
 
+    - name: Get specific of CMake, Ninja
```
Comment (dzenanz) [inline .github/workflows/build-test-package.yml:42]: specific of -> specific version of

File: .github/workflows/build-test-package.yml [hunk 2]
```diff
@@ -169,11 +172,18 @@ jobs:
...
+          - itk-python-git-tag: "v5.1.2"
 
     steps:
     - uses: actions/checkout@v2
 
+    - name: 'Specific XCode version'
+      run: |
+        sudo xcode-select -s "/Applications/Xcode_11.7.app"
+
+    - name: Get specific of CMake, Ninja
```
Comment (dzenanz) [inline .github/workflows/build-test-package.yml:184]: specific of -> specific version of

File: .github/workflows/build-test-package.yml [hunk 3]
```diff
@@ -198,10 +208,15 @@ jobs:
       matrix:
         python-version-minor: [6, 7, 8, 9]
         include:
-          - itk-python-git-tag: "v5.1.1.post1"
+          - itk-python-git-tag: "v5.1.2"
 
     steps:
+    - name: Get specific of CMake, Ninja
```
Comment (dzenanz) [inline .github/workflows/build-test-package.yml:214]: specific of -> specific version of

**Review by dzenanz — COMMENTED** [review]
Looks good

File: .github/workflows/build-test-package.yml (hunk 1 above)
Comment (thewtex) [reply .github/workflows/build-test-package.yml:42]: :+1: fixed

**Pull request merged**